rest-server/core/src/main/scala/com/actionml/core/backup/FSMirror.scala

```diff
@@ -129,4 +114,27 @@ class FSMirror(mirrorContainer: String, engineId: String)
     }
     Valid(jsonComment("Job created to import events in the background."))
   }
+
+  private def importFromFile(file: File, engine: Engine): Long = {
+    var eventsProcessed = 0
+    val src = Source.fromFile(file)
+    try {
+      src.getLines().sliding(512, 512).foreach { lines =>
```

Comment:
@pferrel do we need that setting?

Reply:
No idea. What is this setting based on? Does the `Source.fromFile` library not calculate a reasonable default? Did it speed up the import to set this `(512, 512)`???